sfall/Modules/Tiles.cpp

```diff
@@ -28,6 +28,29 @@
 namespace sfall
 {
 
+#define _OBJ_TYPE_TILE	        (4)
```

Comment:
It's already defined in Enums.h, no need to make another define for it.

Reply:
in the code can not be used enum, compilation error.
maybe it's because of my studio.

Reply:
You need to add "using namespace fo;" at line 30.